File: test_register.dart
import 'dart:convert';
import 'package:http/http.dart' as http;

void main() async {
  final url = Uri.parse('https://astroboomin.co/api/register.php');
  final response = await http.post(
    url,
    headers: {'Content-Type': 'application/json'},
    body: json.encode({
      'name': 'Test Chef',
      'email': 'testchef123@example.com',
      'password': 'password123',
      'role': 'chef',
      'kitchen_name': 'Test Kitchen',
      'phone': '0812345678',
      'avatar': 'http://example.com/avatar.jpg',
      'kitchen_avatar': 'http://example.com/kitchen.jpg',
      'kitchen_cover': 'http://example.com/cover.jpg'
    })
  );
  print('Status code: \${response.statusCode}');
  print('Body: \${response.body}');
}

File: test_register.php
<?php
$data = array(
    'name' => 'Test Chef',
    'email' => 'testchef3@example.com',
    'password' => '123456',
    'role' => 'chef',
    'kitchen_name' => 'Test Kitchen',
    'phone' => '0812345678',
    'avatar' => 'http://example.com/avatar.jpg',
    'kitchen_avatar' => 'http://example.com/kitchen.jpg',
    'kitchen_cover' => 'http://example.com/cover.jpg'
);

$options = array(
    'http' => array(
        'header'  => "Content-type: application/json\r\n",
        'method'  => 'POST',
        'content' => json_encode($data),
        'ignore_errors' => true
    )
);

$context  = stream_context_create($options);
$result = file_get_contents('https://astroboomin.co/api/register.php', false, $context);

echo "Result: " . $result;
?>
